### ISSUE TITLE
Null Pointer using CrC Item Mover

### ISSUE THREAD

**Issue by metnavman** [ReikaKalseki/Reika_Mods_Issues#1295]
http://pastebin.com/34ySkSaZ

Attempting to pull cobblestone out of an ME System. Cobblestone is loaded into the Item Mover correctly. Shift+click on an ME Crafting Interface hooked into the ME Network errors out with the linked log. Going to attempt same by moving the Storage cell to an ME chest and seeing if the problem replicates there, as I remember being able to directly move things in the network a couple versions of CrC back (and that's how the Item Mover is supposed to function anyways, if memory serves).

Edit: All attempts to interact with AE stuff and the Item Mover are resulting in a Null Pointer. Bummer!

**Comment by ReikaKalseki** [comment]
This used to work...

**Comment by ReikaKalseki** [comment]
Does this happen reliably?

**Comment by metnavman** [comment]
I think I figured it out, because it is NOW working, because I gave it a number to move. However, the Lexicon mentions that it will move "up to 10,000" of an item, and at some point you changed it to have that input panel that will allow up to 99999 of an item to be moved. This reliably works, moving 99999 of anything simply by adding it to the panel and performing the required action on an ME Crafting Interface.

The crash happened the very first time I used the Item Mover, before it had any numbers associated with it. Very strange, but it certainly works now, provided it has a number input from 1-99999.
